packages/components/psammead-brand/src/__snapshots__/index.test.jsx.snap

```diff
@@ -89,6 +110,13 @@ exports[`Brand should render correctly 1`] = `
           aria-hidden="true"
           className="c5"
           focusable="false"
+          height={
```

Comment:
:O Styled components is trying to render the height as a html attribute :O

Reply:
If you pass it though as `svgHeights` instead of `height` it might stop this

Reply:
it does :)